tests/containers/podman_pods.pm

```diff
@@ -42,7 +43,7 @@ sub run {
         assert_script_run('podman play kube --down hello-kubic.yaml');
     }
 
-    unless (is_sle('<=15-SP2')) {
+    unless (is_sle('<=15-SP2') || !check_min_runtime_version('4.4.2')) {
```

Comment:
maybe we can remove the check `is_sle('<=15-SP2')` completely, so we only depend on the podman version?

Reply:
ok